src/ontology/imports/ontology-metadata.owl

```diff
@@ -1,573 +1,839 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="http://purl.obolibrary.org/obo/genepio/imports/ontology-metadata.owl#"
+<Ontology xmlns="http://www.w3.org/2002/07/owl#"
```

Comment:
Is this file saved in rdf/xml format?  Try saving it in that format and the diff should match better I bet.

Reply:
This was saved as `OWL/XML Syntax`. I'll resave it as `RDF/XML Syntax` and push.

Reply:
changes pushed to PR